Formula/ffsend.rb

```diff
@@ -0,0 +1,24 @@
+class Ffsend < Formula
+  desc "Fully featured Firefox Send client"
+  homepage "https://gitlab.com/timvisee/ffsend"
+  url "https://github.com/timvisee/ffsend/archive/v0.2.30.tar.gz"
+  sha256 "53130e6b50c6b14c8ec04d6d4077800a0b1855ffd0b2bdbb475ae8ec208695bd"
+
+  depends_on "rust" => [">= 1.32", :build]
+  depends_on "openssl"
+
+  def install
+    # Ensure that the `openssl` crate picks up the intended library.
+    # https://docs.rs/openssl/0.10.19/openssl/#manual
+    ENV["OPENSSL_DIR"] = Formula["openssl"].opt_prefix
+
+    system "cargo", "install", "--root", prefix, "--path", "."
+  end
+
+  test do
+    system "#{bin}/ffsend"
+
+    (testpath/"file.txt").write("test")
+    system "#{bin}/ffsend", "del", shell_output("#{bin}/ffsend upload -q #{testpath}/file.txt")
```

Comment:
What does the test do?

Reply:
It uploads a dummy file through the tool, to delete it immediately afterwards.

It's fairly simple, but it ensures things like OpenSSL bindings are working.

Reply:
Yes, what Tim said. It uploads file.txt, gets its url and then delete the file by that url.

Reply:
Could we check the output of `ffsend del` too? Does it actually output anything we could match with `assert_match`?

Reply:
It only outputs `File deleted` in green, but I think this check is obsolete. If file deletion failed (due to it not existing, or whatever else) it will return with a non-zero exit code.

Reply:
@matevzmihalic Was just thinking, maybe an explicit `-I` flag would be a good idea for both the `upload` and `delete` command. It ensures no interaction is possible, so any unexpected prompts won't show up.
See: https://github.com/timvisee/ffsend#scriptability

Reply:
I added the `-I` flag, but couldn't add `-i` because then it wanted `Owner token` that I did not get from quiet output.

I added `assert_match` for the output of last command.

Reply:
Of course, you're right. For deletion an owner token js required which it automatically provides through the history. Just the I flag should be fine.